src/state-summary/state-card-display.ts

```diff
@@ -0,0 +1,121 @@
+import "@polymer/iron-flex-layout/iron-flex-layout-classes";
+import type { HassEntity } from "home-assistant-js-websocket";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { attributeClassNames } from "../common/entity/attribute_class_names";
+import { computeDomain } from "../common/entity/compute_domain";
+import { computeStateDisplay } from "../common/entity/compute_state_display";
+import { computeRTL } from "../common/util/compute_rtl";
+import "../components/entity/state-info";
+import { UNAVAILABLE_STATES } from "../data/entity";
+import "../panels/lovelace/components/hui-timestamp-display";
+import { haStyle } from "../resources/styles";
+import type { HomeAssistant } from "../types";
+
+@customElement("state-card-display")
+export class StateCardDisplay extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj!: HassEntity;
+
+  @property({ type: Boolean }) public inDialog = false;
+
+  // property used only in CSS
+  @property({ type: Boolean, reflect: true }) public rtl = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div class="horizontal justified layout">
+        <state-info
+          .hass=${this.hass}
+          .stateObj=${this.stateObj}
+          .inDialog=${this.inDialog}
+        >
+        </state-info>
+        <div class="${this.computeClassNames(this.stateObj)}">
```

Comment:
Use a `classMap` here

Reply:
What would we gain then? For `classMap` I would have to duplicate the logic that the common helper `attributeClassNames` already carries. I however, cleaned up its usage now.